src/vision/status_quo/alan_has_an_event_loop.md

```diff
@@ -0,0 +1,104 @@
+# 😱 Status quo stories: Alan has an external event loop and wants to use futures/streams
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+
+As a first Rust Project, Alan decides to program his own IRC Client.
+
+Since it is Alans first Project in Rust, it is going to be a private one. He is going to use it on is Mac, so he decides to go with the cocoa crate to not have to learn any Framework specific quirks. This way Alan can get a feel of Rust itself.
+
+### Alans hopes and dreams
+Despite a learning curve, he managed to creating a first window and have some buttons and menus works. After the initialisation is done, the App hand over control to [CFRunLoop::Run](https://developer.apple.com/documentation/corefoundation/1542011-cfrunlooprun?language=occ).
+
+Once Alan is happy whit his Mock UI, he wants to hook it up with some actual features. He is happy to learn that Rust has Features he already knows.
+* Promises => Futures
+* Observables => Streams.
+
+Alan smiles, thinking he knows what and more importantly how to do this.
+
+### First time dealing with runtimes
+
+Unfortunately, coming from frameworks like Angular or Node.js, Alan is not used that he himself is responsible for driving the progress of Async/Streams. 
```

Comment:
```suggestion
Unfortunately, coming from frameworks like Angular or Node.js, Alan is not used that being responsible for diving the progress of Futures/Streams. 
```

Does he first try to write some code and then find that...nothing happens? Or does he just figure that out from reading stuff I guess?